Fixes for 1.11

replaceafill opened the pull request:
A few fixes from recent changes to the `qa` branches:

- Update dropdown option in backlog
- Fix allow-orphaned-key-delete tag
- Drop PREMIS 2 namespace

Connected to https://github.com/archivematica/Issues/issues/820

ross-spencer reviewed (approved):
Perfect! :+1:

ross-spencer commented:
(Whoops - noticed it's still WIP - how did I miss that?! (general excitement for testing I think!)) (Okay, well, the initial PREMIS changes work well, I don't think I can retract approval, but happy to go back over this again when complete!)

replaceafill commented:
@ross-spencer Thanks! I'm just waiting for Jenkins to pass the `encrypt` tags.

replaceafill committed:
Fixes for 1.11

- Update dropdown option in backlog
- Fix allow-orphaned-key-delete tag
- Drop PREMIS 2 namespace
